core/chaincode/shim/chaincode.go

```diff
@@ -958,44 +958,37 @@ func (c *ChaincodeLogger) IsEnabledFor(level LoggingLevel) bool {
 	return c.logger.IsEnabledFor(logging.Level(level))
 }
 
-// Note: We're only creating the 'f' forms of the logging functions here for
-// consistency with Go language conventions around formatted I/O routines, and
-// to avoid confusion with the conventions used in the core code. It is
-// possible that some day the core code will also change from using
-// logger.Debug() to logger.Debugf() etc., and we want to protect chaincode
-// writers from that hiccup if it occurs.
-
 // Debugf logs will only appear if the ChaincodeLogger LoggingLevel is set to
 // LogDebug.
 func (c *ChaincodeLogger) Debugf(format string, args ...interface{}) {
```

Comment:
Having this wrapper seems unnecessarily redundant now. Not necessarily suggesting that we change it in this PR, but I will create an issue for subsequent fix.